addons/mail/static/src/models/message/message.js

```diff
@@ -233,8 +233,8 @@ function factory(dependencies) {
                         messageSeenIndicators: [[
```

Comment:
should probably take the opportunity to change this here, no need to update thread, it's computed from the inverse relation